README.md

```diff
@@ -29,15 +29,23 @@ This will fetch and parse the entire blockchain.
 Configuration file: `/conf.json`
 
 Basic settings:
-```json
+```javascript
 {
+    // Your node API RPC address.
     "nodeAddr":     "localhost",
+    // Your node API RPC port.
     "gethPort":     8545,
+    // This is the start block of the blockchain, should always be 0 if you want to sync the whole ETC blockchain.
     "startBlock":   0,
+    // This is usually the 'latest'/'newest' block in the blockchain, this value gets updated automatically, and will be used to patch missing blocks if the whole app goes down.
     "endBlock":     "latest",
+    // Prints out the log of what it is doing, Might not work for all messages in this release
     "quiet":        true,
+    // If this is set to true at the start of the app, the sync will start syncing all blocks from lastSync, and if lastSync is 0 it will start from whatever the endBlock or latest block in the blockchain is.
     "syncAll":      true,
+    // If set to true and below value is set, sync will iterated through the # of blocks specified
     "patch":        true,
+    // If `patch` is set to true, the amount of block specified will be check from the latest one.
     "patchBlocks":  100,
     "blocks": [ {"start": 2000000, "end": "latest"}],
```

Comment:
If your machine running explorer goes down. If patch is set to true it will go over however many last blocks you specified to makes sure any blocks weren't missed during down time.